Source/FolioReaderContainer.swift

```diff
@@ -185,3 +199,23 @@ open class FolioReaderContainer: UIViewController {
         return isNight(.lightContent, .default)
     }
 }
+
+extension FolioReaderContainer {
+    func alert(message: String) {
```

Comment:
It will show user what is the error and then revert back application instead of getting crash.